priv/gettext/vi/LC_MESSAGES/default.po

```diff
@@ -0,0 +1,74 @@
+msgid ""
+msgstr ""
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Project-Id-Version: Torch\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: https://github.com/mojotech/torch\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: en\n"
```

Comment:
@nhphuc412 I believe this is incorrect.  This isn't an "en" (English) translation file.  If you can update this value to the proper locale entry for the included Vietnamese translations and update this commit we can get this merged in and a new release pushed out.

Thanks.

Reply:
@cpjolicoeur I updated value for vietnamese translations